### DISCUSSION TITLE
Building willow

### DISCUSSION THREAD

**Discussion by MOJNICK** [toverainc/willow#318] [Q&A] [answered]
I'm trying to follow instructions on https://heywillow.io/development/ , have no success with either main or release/v0.1 branch.

I get an error when building:
`requests.exceptions.SSLError: HTTPSConnectionPool(host='192.168.100.6', port=8502): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)')))`

I have tried to set different urls:
wss://192.168.100.6:8502
ws://192.168.100.6:8502
http://192.168.100.6:8502
, nothing works.

Is it expected?

**Comment by nikito** [comment, marked as answer]
Url is ws://192.168.100.6:8502/ws
You need the /ws at the end 🙂
You can also use wss://192.168.100.6:8502/ws
Also I assume you are referring to the was url here, in connection settings?

**Reply by MOJNICK** [reply]
Thank you - exactly that was the problem.
I'm again on the way to build Willow for non standard (esp32-s3 + i2s mic)  :)
